Marlin/Configuration.h

```diff
@@ -230,9 +250,9 @@ Here are some standard links for getting your machine calibrated:
 // If your configuration is significantly different than this and you don't understand the issues involved, you probably
 // shouldn't use bed PID until someone else verifies your hardware works.
 // If this is enabled, find your own PID constants below.
-//#define PIDTEMPBED
```

Comment:
We should keep this on now, but ultimately we should test to see if this is actually needed